lib/l10n/arb/app_en.arb

```diff
@@ -0,0 +1,82 @@
+{
+"language":"English",
+"@language":{
+  "description":"The current Language"
+},
+
+"closeIcon":"close icon",
```

Comment:
you can remove this as we removed `closeIcon` from `ValueKey`

Reply:
Yes got it.